docusaurus/docs/cloud/advanced/email.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Email Provider
+displayed_sidebar: cloudSidebar
+description: Configure Strapi Cloud to use a third-party email provider.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/email.html
+---
+
+# Email Provider
+
+Strapi Cloud comes with a basic email provider out of the box. However, you can also configure it to utilize a third-party email provider, if needed.
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.2+`.
+- Credentials for a third-party email provider (see [Strapi Market](https://market.strapi.io/)).
+
+:::
+
+## Configuration
+To configure a third-party email provider, in your Strapi project, create a `./config/env/production/plugins.js` or `./config/env/production/plugins.ts` file with the following content:
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  Required Email Provider Plugin Configuration
+  // ...
+});
+```
+Each provider will have different configuration settings available. Review the respective entry for that provider in the [Marketplace](https://market.strapi.io/).
+
+Below are example configurations for the Email plugins.
+
+<Tabs groupId="email-examples" >
```

Comment:
In the examples, you can add a title to precise the name of the file where the content should be. We do that in most examples in the Dev Docs, I think it could be nice to do the same here as well 😊 [Here's an example](https://docs.strapi.io/dev-docs/configurations/database#configuration-examples) and [here's the part of the style guide that explains how to add a title](https://www.notion.so/strapi/Strapi-Documentation-Style-Guide-b9c21387732741ed8b0a3f9701612577?pvs=4#4f6204d506de4329810d5e3e1a45040d).

Reply:
It's also possible to add another level of tabs to have one for JS and one for TS. @pwizla Do you usually use the JS/TS subtabs in the examples or in the config code block?

Reply:
Great idea about titles and sub-tabs, Mégane. Please see my comment above for more details, Kellen.